src/index.css

```diff
@@ -411,3 +411,8 @@ div.monokleEditorErrorRefGlyphClass.monokleEditorUnsatisfiedRefGlyphClass::after
 .filters-preset-dropdown .ant-select-item-option-content:hover .anticon {
   display: block;
 }
+
+/*temporary*/
+.libes__A6-dev__sc-akj29i-8.gOklpn:last-child {
```

Comment:
I have asked the question about hiding or removing the left menu option under Razvan's Comment , in case of hiding this is actual, in case if we remove left navigation button completely, this should be removed too. So in upper question, we can have answer to this as well

Reply:
Removing it

Reply:
Done